.lintrunner.toml

```diff
@@ -338,6 +338,26 @@ init_command = [
 ]
 is_formatter = true
 
+[[linter]]
+code = 'SQLFLUFF'
+# include_patterns = ['torchci/clickhouse_queries/**/*.sql']
+include_patterns = ['torchci/clickhouse_queries/workflow_load/query.sql']
+exclude_patterns = [
+]
+command = [
+    'python3',
+    'tools/linter/adapters/sqlfluff_linter.py',
+    '@{{PATHSFILE}}',
+]
+init_command = [
+    'python3',
```

Comment:
The lint workflow complains about this https://github.com/pytorch/test-infra/actions/runs/12956209088/job/36141963692?pr=6209#step:4:243
```suggestion
```